Elochukwu Gerald Okafor/laptop-model-faizal-update.ts

```diff
@@ -0,0 +1,180 @@
+/** NOTE: I had to change a few things to suit my implementation better */
+
+// An 'updatable' interface means that the component is updatable
+interface Updatable<T> {
```

Comment:
I like this stuff, nice work Elo.
You can share it with your guys on the group chat.
I will also want you to share the materials you used for it.

Nice stuff

Reply:
Ok, thanks sir. I didn't use any materials tho, this is what came to mind.